app/javascript/dashboard-syncing-icon.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="loading-icon" v-if="loading"></div>
+</template>
+
+<script>
+/* eslint-disable no-alert */
+import axios from 'axios';
+
+export default {
+  props: ['id'],
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  methods: {
+    checkSync() {
+      const TIMEOUT = 5000;
+      axios.get(`/api/organizations/${this.id}/check_sync`)
+        .then((res) => {
+          console.log(this.loading);
+          if (res.data.state === 'completed') {
+            if (this.loading === true) {
+              location.reload(true)
+            }
+            this.loading = false;
+          } else if (res.data.state === 'executing') {
+            this.loading = true;
+            setTimeout(() => { this.checkSync() }, TIMEOUT);
```

Comment:
This line has 2 statements. Maximum allowed is 1  max-statements-per-line<br>Missing semicolon                                 semi